diffs/models.py

```diff
@@ -105,8 +105,11 @@ def zadd(self, members, score=1):
         else:
             for member, score in members.items():
                 _members += [member, score]
-
-        return self.db.zadd(self.key, *_members)
+        result = self.db.zadd(self.key, *_members)
```

Comment:
I couldn't find a way to include ttl in `zadd`, but the python library's got a [`setex(...)`](http://redis-py.readthedocs.io/en/latest/#redis.Redis.setex) function that includes a parameter for seconds to expire. Should I use that instead of `zadd` and `expire`?